src/App.js

```diff
@@ -249,12 +252,24 @@ class App extends Component {
   handleNavigatorRef = navigatorRef =>
     Navigation.setTopLevelNavigator(navigatorRef);
 
-  handleTransactionConfirmed = () => {
-    logger.log('Reloading all data from L2 explorers in 10!');
-    setTimeout(() => {
-      logger.log('Reloading all data from L2 explorers NOW!');
-      store.dispatch(explorerInitL2());
-    }, 10000);
+  handleTransactionConfirmed = tx => {
+    logger.log('reloading explorer data in 10');
+    const network = ethereumUtils.getNetworkFromChainId(tx.chainId);
+    const isL2 = isL2Network(network);
+    setTimeout(
+      () => {
+        if (isL2) {
+          logger.log('Reloading all data from L2 explorers!');
+          store.dispatch(explorerInitL2());
```

Comment:
is it worth only reloading the specific L2 we need to check or nah?

Reply:
nah

Reply:
skylar i like ur thinking tho

🤨 bruno over here acting like we rich in compute units 😛